### ISSUE TITLE
[BUG] unable to login / submit local Immich server 

### ISSUE THREAD

**Issue by cktechie2012** [giejay/Immich-Android-TV#83]
**Describe the bug**

**To Reproduce**
1. Go to '...' login prompt, login to Immich Server using IP address
3. Click on '....' Entered thr API
4. Scroll down to '....' Click submit 
5. See error - Please enter a valid hostname

**Expected behavior**
Should have logged in 

**Screenshots**

**My setup (please complete the following information):**
 - Android OS version:  
![IMG_6315](https://github.com/user-attachments/assets/01019168-c600-4d7a-8da5-80eef37148b5)

 - Device: Google TV with Chromecast  & Verizon Stream TV Box
 - Immich app version [e.g. 1.10] 2.4.1

**Additional context**

**Comment by cktechie2012** [comment]
Androis TV VERSION 12
KERNEL 5.4.242

**Comment by jfroy525** [comment]
Having the same issue

**Comment by giejay** [comment]
Are you including http? For me it works fine, both locally and remote. 

example: http://192.168.2.114:2283

**Comment by cktechie2012** [comment]
thanks for the response. 

I was able to login with the URL. The description on login page is a bit misleading.

Would recommend a change to show it as "Enter the URL in the format http://<ip-address>:2283 -- or -- include http and port on the backend after the IP address is included.

Though seeing a very slow response, as I have 60K+ photos & 5K+ videos

**Comment by giejay** [comment]
Where are you seeing a slow response? All the API calls are paginated so the backend size should not matter

**Comment by cktechie2012** [comment]
upon login, when I try to browse on the left side menu structure, the application kind of freezes my devices - seeing this happen on both Google TV with Chromecast as well as Stream TV box.

**Comment by giejay** [comment]
And you don't have this issue when using the demo server?

**Comment by giejay** [comment]
Can you try to disable all views and re enable them one by one? To see if it's a specific album issue for example.

**Comment by cktechie2012** [comment]
The albums are the first on the list, and videos dont seem to play and slows down the app as well as google tv. If i try to view regular photos, it works well

**Comment by giejay** [comment]
But it's also slow for the demo server?

**Comment by rpardee** [comment]
I am having this issue also. Using a google tv streamer 4k, I am unable to log in via username/pwd using the scan-QR-code function.  I continually get an 'Invalid hostname!' error. Can you help?

<img width="806" height="1795" alt="Image" src="https://github.com/user-attachments/assets/0cd1142d-161b-4745-bac7-f0e5c37f4681" />

**Comment by rpardee** [comment]
For some reason, logging in via an API key (with the same exact server URL) worked for me.

> I am having this issue also. Using a google tv streamer 4k, I am unable to log in via username/pwd using the scan-QR-code function. I continually get an 'Invalid hostname!' error. Can you help?
> <img alt="Image" width="806" height="1795" src="https://private-user-images.githubusercontent.com/26327/586670946-0cd1142d-161b-4745-bac7-f0e5c37f4681.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Nzc2ODM1MzMsIm5iZiI6MTc3NzY4MzIzMywicGF0aCI6Ii8yNjMyNy81ODY2NzA5NDYtMGNkMTE0MmQtMTYxYi00NzQ1LWJhYzctZjBlNWMzN2Y0NjgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA1MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNTAyVDAwNTM1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg0OGNkYzljMzZmOGVkN2FmZjNkY2RmM2JlN2M2ZGY5NTBhZTUzZmIzMDE3YjRjYjc5ZGYzZTU3NGEyYzNiZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.tVY2ouHmPJPYRosIztnKWzFUcAFPNzWcd3F1Gm0KfUU">

**Comment by giejay** [comment]
Yeah I should make this more clear in the UI. The login using qr is only usable for external hosted immich instances. My login controller needs to connect to your host and that of course doesn't work using a local IP.
